### PR TITLE
Added tags for branches and discord bot works (without pings)

### PR DESCRIPTION
**Pull request by FishOfPitt116**

### CODE REVIEW

**Comment by FishOfPitt116** [comment]
**Bug Found:** Tag that contains no posts will not have a tag-specific posts page, so GET request for said page will lead to error. Not site breaking, but it would be cool to have a placeholder site and something that says "go back home" instead of just crashing lol

File: .eleventy.js [hunk 1]
```diff
@@ -21,6 +21,14 @@ module.exports = function (eleventyConfig) {
 
   eleventyConfig.addCollection("tagList", function (collection) {
     let tagSet = new Set();
+    tagSet.add("game dev");
+    tagSet.add("panther ml");
+    tagSet.add("pitt cyber security");
```
Comment (agrattan0820) [inline .eleventy.js:26]: Thoughts on removing "pitt" from "pitt cyber security"?

Comment (agrattan0820) [reply .eleventy.js:26]: So that the scss class can just be "cyber"?

File: src/sass/_mixins.scss [hunk 2]
```diff
@@ -33,6 +33,38 @@
...
+
+  .game {
+    background-color: $fuschia;
+  }
+
+  .panther {
+    background-color: $green;
+  }
+
+  .pitt {
```
Comment (agrattan0820) [inline src/sass/_mixins.scss]: Change to .cyber?

File: src/sass/_mixins.scss [hunk 3]
```diff
@@ -71,6 +103,38 @@
...
```
Comment (agrattan0820) [inline src/sass/_mixins.scss]: Same .cyber change

**Review by agrattan0820 — COMMENTED** [review]
Looks good besides changing from "pitt cyber security" to "cyber security"

**Comment by agrattan0820** [comment]
> **Bug Found:** Tag that contains no posts will not have a tag-specific posts page, so GET request for said page will lead to error. Not site breaking, but it would be cool to have a placeholder site and something that says "go back home" instead of just crashing lol

We can use this https://www.11ty.dev/docs/quicktips/not-found/

**Pull request merged**